site/docs/develop/java.md

```diff
@@ -210,18 +210,46 @@ The documentation for how to configure Nessie server authentication can be found
 The `BasicAuthenticationProvider` allows connecting to a Nessie server that has `BASIC` authentication enabled.
 Note that `BASIC` is not supported in production and should only be used for development/testing.
```

Comment:
I believe servers (outside of unit tests) do not allow "basic" auth at all. Cf. this in `applicaiton.properties`:

```
# fixed at buildtime
quarkus.http.auth.basic=false
```

Would you mind fixing this doc paragraph too?

Reply:
I think we can remove references to `BASIC` auth from all docs at this point. WDYT?